.github/workflows/portal-checks.yml

```diff
@@ -34,7 +34,9 @@ jobs:
     - name: Install Dependencies
       run: |
         python3 -m pip install --upgrade pip setuptools
-        pip install -r requirements-dev.lock
+        pip install poetry
```

Comment:
Can this happen with the previous line as well? Don't know enough to say myself

Reply:
Yes, I think so.

Reply:
Oh interesting, did we not need to install pip at all? Already there?